lego/apps/email/tasks.py

```diff
@@ -0,0 +1,23 @@
+from structlog import get_logger
+
+from lego import celery_app
+from lego.apps.email.notifications import WeeklyNotification
+from lego.apps.users.models import AbakusGroup
+from lego.utils.tasks import AbakusTask
+
+log = get_logger()
```

Comment:
This seems to not be used, but I now see that we have these things laying around everywhere in our codebase, so I'll let it slide. You never know when you'll need it, I suppose?

Reply:
That is a good question. I just copied it from another task :shrug: . I could look into it if you want